transport/src/main/java/io/scalecube/transport/TransportImpl.java

```diff
@@ -218,6 +218,28 @@ private TransportImpl onBind(DisposableServer server) {
     return new TransportImpl(address, server, networkEmulator, this);
   }
 
+  public Mono<Message> requestResponse(final Message request, Address address) {
+    return Mono.create(s -> {
+      Objects.requireNonNull(request);
+      Objects.requireNonNull(request.correlationId());
+      Objects.requireNonNull(request.sender());
```

Comment:
Objects.requireNonNull(request, "request must be not null");
Objects.requireNonNull(request.correlationId(), "correlationId must be not null");
Objects.requireNonNull(request.sender(), "sender must be not null");

Reply:
fixed